zerver/data_import/slack_message_conversion.py

```diff
@@ -215,10 +217,12 @@ def replace_links(text: str) -> str:
 def render_block(block: WildValue) -> str:
     # https://api.slack.com/reference/block-kit/blocks
     block_type = block["type"].tame(
-        check_string_in(["actions", "context", "divider", "header", "image", "input", "section"])
+        check_string_in(
+            ["actions", "context", "divider", "header", "image", "input", "section", "rich_text"]
+        )
     )
-    if block_type == "actions":
-        # Unhandled
+    unhandled_types = ["actions", "rich_text"]
```

Comment:
"rich_text" block type is basically the same as the "text" field, so this is filtered out to avoid duplication